Components/MovieList.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect } from 'react';
+
+import { LinearGradient } from 'expo-linear-gradient';
```

Comment:
Since `LinearGradient` is outer dependency, it should go to part of imports reserved for outer dependencies (just below `React`)